lib/sponsors/guardian.ex

```diff
@@ -1,6 +1,10 @@
 defmodule Sponsors.Guardian do
   use Guardian, otp_app: :sponsors
 
+  def resource_from_claims(%{"sub" => "user:" <> user_id}) do
+    {:ok, user_id}
```

Comment:
I'm not 100% sure if the code needs this `user_id` to be a string or integer, or if it will _just work_ :tm: 

@doomspork

Reply:
A string should be fine. We don't do anything special with this here or elsewhere :+1: